.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
-# This should match the owning team set up in https://github.com/orgs/opensearch-project/teams
-*   @opensearch-project/anomaly-detection
\ No newline at end of file
+* @jmazanec15 @jngz-es @kaituo @wnbts @saratvemulapalli @ohltyler @vamshin @VijayanB @ylwu-amzn
+
```

Comment:
Could you add @amitgalitz @sudiptoguha @jackiehanyang @sean-zheng-amazon and remove @wnbts (who isn't member of the org anymore) ?

Reply:
Let's do the exercise in adding other maintainers after the baseline exercise. All yours.